### ISSUE TITLE
I can't launch Goverlay 0.6.4

### ISSUE THREAD

**Issue by Alfenstein8** [benjamimgois/goverlay#156]
I can launch Goverlay 0.6.4 with the terminal but then i get a error and I can't se any mangohud overlay in Goverlay.
The error:
```
QWidget::paintEngine: Should no longer be called
paint
```
System info:
```
OS: openSUSE Tumbleweed x86_64 
Kernel: 5.14.14-1-default 
Uptime: 2 hours, 41 mins 
Packages: 2923 (rpm), 31 (flatpak), 9 (snap) 
Shell: bash 5.1.8 
Resolution: 2560x1440, 1920x1080, 1080x1920 
DE: Plasma 5.23.2 
WM: KWin 
WM Theme: Breeze 
Theme: Breeze [Plasma], Breeze [GTK2/3] 
Icons: [Plasma], breeze-dark [GTK2/3] 
Terminal: konsole 
CPU: AMD Ryzen 7 2700X (16) @ 3.800GHz 
GPU: NVIDIA GeForce RTX 2070 Rev. A 
Memory: 6043MiB / 32012MiB 
```

**Comment by benjamimgois** [comment]
After the hud integration into goverlay, you have to execute goverlay with mangohud. Try to run it with the commands:

`QT_QPA_PLATFORM=xcb mangohud --dlsym goverlay --style fusion`

The .desktop file should have this new commands, it's already on the AUR (arch/manjaro).

**Comment by SonGoku90** [comment]
Hello
I have the same how to fix it?

**Comment by Alfenstein8** [comment]
> After the hud integration into goverlay, you have to execute goverlay with mangohud. Try to run it with the commands:
> 
> `QT_QPA_PLATFORM=xcb mangohud --dlsym goverlay --style fusion`
> 
> The .desktop file should have this new commands, it's already on the AUR (arch/manjaro).

yeah the .desktop file have it already. When i run the command it says
```
can't get libdl.so
```

**Comment by benjamimgois** [comment]
> libdl.so

What distro are you running ?

**Comment by Alfenstein8** [comment]
> 
> 
> > libdl.so
> 
> What distro are you running ?

OS: openSUSE Tumbleweed x86_64

**Comment by benjamimgois** [comment]
@Alfenstein8 , it seems something related to opensuse Tumbleweed x86_64. I'll make some research on it. The next release will have a new start script implementation, so i guess it will make things better.

**Comment by benjamimgois** [comment]
I'm investigating the issue. more people are reporting the same.

**Comment by Alfenstein8** [comment]
> I'm investigating the issue. more people are reporting the same.

Thank you for looking into it. I appreciate it.

**Comment by lanceturbes** [comment]
Also still having this error on Tumbleweed, using GOverlay 0.6.4 (which is the version currently packaged in OpenSUSE's repository). 

I tried both the direct releases of 0.6.4 and 0.7 from this repository, but the error persists (exactly as described in the original post); the `goverlay` command _will_ open the program (without a mangohud preview), but the start script (and the direct command of `QT_QPA_PLATFORM=xcb mangohud --dlsym goverlay --style fusion`) will _not_ open the program, returning the error message of `can't get libdl.so`.

No rush or complaints, though (I can still use it to cap frame rate and stuff when launched directly); I know you guys are busy fixing the current fusion theme weirdness and other stuff too -- appreciate you looking into this anyways. Good luck!

**Comment by HolySoap** [comment]
@lanceturbes the Tumbleweed version got recently updated to version 0.7.

However @benjamimgois , now I have the same issue with `libdl.so` that I didn't have before with v0.6.4.
Terminal output:
```
$> QT_QPA_PLATFORM=xcb mangohud --dlsym goverlay --style fusion
can't get libdl.so
```

Running under Plasma Wayland.

**Comment by kevinsmia1939** [comment]
In openSUSE Tumbleweed, there is only libdl.so.2, but goverlay seems to look for libdl.so

**Comment by TimeFreeze1337** [comment]
Also having the issue with openSUSE Tumbleweed that it complains about the libsdl.so

**Comment by benjamimgois** [comment]
Hey guys, i took some days off this month, but i'll try to work on the software again now. i have zero knowledge on opensuse, I'll try to take a look, but it would be great if anyone more familiar with the distro could help.

**Comment by kevinsmia1939** [comment]
> Hey guys, i took some days off this month, but i'll try to work on the software again now. i have zero knowledge on opensuse, I'll try to take a look, but it would be great if anyone more familiar with the distro could help.

Hi, I do have openSUSE. 
But from what I can find, openSUSE only have libdl.so.2. I don't know why would it only want libdl.so?

**Comment by Gryxx** [comment]
This is a problem with MangoHud.
https://github.com/flightlessmango/MangoHud/issues/601#issuecomment-932426614

**Comment by DarkWav** [comment]
For me this issue is fixed in openSUSE TW snapshot `20220313`, `mangohud` is now at version `0.6.6.1` and with that goverlay works for me again.

**Comment by bomiyr** [comment]
I also can confirm it's fixed on TW snapshot 20220313

**Comment by benjamimgois** [comment]
Ok, thanks !
